src/views/dashboard/User.tsx

```diff
@@ -2,54 +2,47 @@ import React, { useState, useEffect } from "react";
 import { Query, QueryResult } from "react-apollo";
 import { GET_USER } from "../../graphql-requests/queries";
 import "../home/Home.css";
-import TextField from "@material-ui/core/TextField";
 //import { makeStyles } from "@material-ui/core/styles";
 
+//let user_id = localStorage.getItem("user_id");
+
 const User: React.FC = (props: any) => {
    const [user_id, setUser] = useState("");
+
    useEffect(() => {
       let user_id = localStorage.getItem("user_id");
```

Comment:
There is some ambiguity going on here. I suggest changing the name of this variable, so you aren't overriding the `user_id` component-state variable. This issue for becomes apparent when I consider the user_id subscription you have for the useEffect.